### ISSUE TITLE
YouTube Videos not Playing

### ISSUE THREAD

**Issue by ChiefZack17** [kboghdady/youTube_ads_4_pi-hole#61]
For some reason, YouTube videos have been unable to load since this morning.

https://tricorder.pi-hole.net/1l8hjn93cy

**Comment by Mxtchhel** [comment]
Same issue

**Comment by ChiefZack17** [comment]
Rather than deactivate pi hole, I deactivated this gravity list only to find that the installation added ~20000 blacklisted domains. 

The script they ran manually added them all to the exact blacklist so I entered,

sudo sqlite3 /etc/pihole/gravity.db "delete from domainlist where type=1;"

I also edited
sudo crontab -e
to remove the scheduled update line and deleted the folder from the directory. That seems to have made YouTube functional again.

**Comment by marcinlubojanski** [comment]
I don't have pihole but instead I run dnsmasq with custom hosts file on my padavan enabled router. I have downloaded the list of blacklisted hosts, and added to the hosts with 0.0.0.0 IP address.  Tested with ping, it resolves to localhots now. But if I play youtube vides form mobile (which uses my router as DNS) I still see lot of ads. I didn't use youtube from few days so I doub't the IP addresses had been cached.
